app/utils/markdown/index.ts

```diff
@@ -20,7 +20,7 @@ type LanguageObject = {
 
 // pattern to detect the existence of a Chinese, Japanese, or Korean character in a string
```

Comment:
Can you also update this comment to account for thai?

Reply:
Done